### DISCUSSION TITLE
Προθεσμίες παραδοτέων

### DISCUSSION THREAD

**Discussion by p20sami** [courses-ionio/hci#1604] [Announcements]
Χθές, έγινε ενημέρωση του πίνακα παραδοτέων, όπου αναφέρεται πως η συμπλήρωση της εβδομάδας θεωρείται ως η ημέρα πριν το επόμενο μάθημα. Δεδομένου οτι έχουμε Τρίτη και Παρασκευή, ποια απο τις δύο ημέρες θα θεωρούμε την ημέρα για την οποία η προηγούμενη της είναι η ημέρα που λήγει η διορία? Επίσης σχετικά με τα παραδοτέα και τις διορίες, λόγω του οτι την 1η εβδομάδα (11-17 Οκτωβρίου) δεν έγιναν τα μαθήματα, δεν γνωρίζαμε το πότε πρέπει να συνταχθεί το 1ο παραδοτέο διότι δεν υπήρχε και η σχετική ενημέρωση κάτω απο τον πίνακα τότε. Ως εκ τούτου, μέχρι την προηγούμενη ημέρα της επόμενης μας πλέον συνάντησης (ημέρα που θα μας υποδείξετε στην ερώτηση αυτή), θα πρέπει να συντάξουμε το 1ο (δηλαδή έχουμε ακόμη χρόνο για το 1ο) ή το 2ο παραδοτέο? Διότι αναφέρετε πως όποιος χάσει το 1ο και το τελευταίο παραδοτέο βαθμολογείται με άριστα την βάση, αλλά λόγω των προαναφερθέντων δε γνωρίζαμε το πότε να συνταχθεί το 1ο. Εάν χαθεί ένα απο τα 2 αυτά παραδοτέα, βαθμολογούμαστε πάλι με άριστα τη βάση η για να γίνει αυτό πρέπει να χαθούν και τα 2? Δεν δημιούργησα 2 ξεχωριστές συζητήσεις για τα ερωτήματα διότι πιστέυω σχετίζονται άμεσα.

Σας ευχαριστώ εκ των προτέρων.

**Comment by epidrome** [comment]
Όλες οι προθεσμίες είναι ενδεικτικές!

Ο σκοπός των προθεσμιών είναι να έχουμε τακτικά δείγματα γραφής και προόδου ώστε να γίνει σχολιασμός πριν να είναι αργά

Αν χαθούν 2-3 προθεσμίες για λίγες μέρες δεν πειράζει, ειδικά αν αυτό γίνει για να πετύχετε καλύτερη ποιότητα

Το 1ο παραδοτέο δείχνει πότε άρχισε να ασχολείται κάποιος με το μάθημα και προφανώς αν άρχισε να ασχολείται τον Δεκ είναι διαφορετικό με το αν άρχισε τον Οκτ

**Reply by p20sami** [reply]
Κατανοητό, σας ευχαριστώ.
